go.mod

```diff
@@ -26,11 +26,11 @@ require (
 	github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46
 	github.com/andygrunwald/go-gerrit v0.0.0-20190120104749-174420ebee6c
 	github.com/aws/aws-k8s-tester v0.0.0-20190114231546-b411acf57dfe
-	github.com/aws/aws-sdk-go v1.23.20
+	github.com/aws/aws-sdk-go v1.23.22
 	github.com/bazelbuild/buildtools v0.0.0-20190917191645-69366ca98f89
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/bwmarrin/snowflake v0.0.0
-	github.com/clarketm/json v1.13.0
+	github.com/clarketm/json v1.13.1
```

Comment:
Can we get rid of this dep @clarketm ?

Reply:
No, it is required by [`genyaml`](https://github.com/kubernetes/test-infra/blob/a3fd5b74f9e21e93efe077212ebd0735bf4820b9/pkg/genyaml/genyaml.go#L75) (at least until I can create a custom encoder ⌚️).

Reply:
https://github.com/kubernetes/test-infra/blob/7b736ada2fa76307bf3c5921737936dcfb161957/prow/config/config.go#L470

https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/duration.go

Reply:
My take is that since it takes an interface, it can handle *any* arbitrary type. IMO, it would be impractical to use this tool if a user had to create a custom marshaller for every `struct` the source file(s). It would complicate it and reduce its utility. I am not saying that *this*/*my* approach is the best way, but it does make the tool much easier to use.

https://github.com/kubernetes/test-infra/blob/a3fd5b74f9e21e93efe077212ebd0735bf4820b9/pkg/genyaml/genyaml.go#L411

Reply:
So basically you're just reinventing http://sigs.k8s.io/yaml? Please get rid of this.

Reply:
This would not work with `http://sigs.k8s.io/yaml` because it it outdated and does not support the [`Node`](https://github.com/kubernetes/test-infra/blob/a3fd5b74f9e21e93efe077212ebd0735bf4820b9/pkg/genyaml/genyaml.go#L423) type which it critical to the comment-injection logic. 

xref: https://github.com/kubernetes-sigs/yaml/issues/18

Reply:
I am not opposed to switching to `http://sigs.k8s.io/yaml` using it once these features are supported (assuming it exposes these built in `go-yaml`/`v3` types and properly filters zero-values structs)